cpp/include/cuvs/core/c_api.h

```diff
@@ -141,7 +141,8 @@ cuvsError_t cuvsRMMFree(cuvsResources_t res, void* ptr, size_t bytes);
  * @return cuvsError_t
  */
 cuvsError_t cuvsRMMPoolMemoryResourceEnable(int initial_pool_size_percent,
-                                            int max_pool_size_percent);
+                                            int max_pool_size_percent,
+                                            int managed);
```

Comment:
I think this should be a bool - and we should also add this managed parameter to the docstring above

```suggestion
                                            bool managed);
```